app/javascript/profileDropdown/flagButton.js

```diff
@@ -0,0 +1,63 @@
+/* eslint-disable no-alert */
+export function initFlag() {
```

Comment:
Could we add some brief documentation for these functions? [`initializeBroadcast.js`](https://github.com/forem/forem/blob/master/app/assets/javascripts/initializers/initializeBroadcast.js) has some good JSDoc examples we could follow! 😃

Reply:
Yeah, sure.